tests/UnifiedSpecTests/Context.php

```diff
@@ -499,7 +499,7 @@ private static function prepareCollectionOrDatabaseOptions(array $options): arra
 
     private static function prepareBucketOptions(array $options): array
     {
-        Util::assertHasOnlyKeys($options, ['bucketName', 'chunkSizeBytes', 'readConcern', 'readPreference', 'writeConcern']);
+        Util::assertHasOnlyKeys($options, ['bucketName', 'chunkSizeBytes', 'disableMD5', 'readConcern', 'readPreference', 'writeConcern']);
```

Comment:
Per my earlier comment about no tests using `bucketOptions`, I doubt this change is required. But if you'd like to keep it, I think it warrants the same comment I suggested below in `Util.php` so any future reader understands why this option is still here.

Reply:
You are correct, this option is not necessary for the existing spec tests. But it should be allowed according to the spec.

```suggestion
        Util::assertHasOnlyKeys($options, ['bucketName', 'chunkSizeBytes', 'readConcern', 'readPreference', 'writeConcern']);
```